modules/internal/DefaultRectangular.chpl

```diff
@@ -1791,6 +1791,10 @@ module DefaultRectangular {
   }
 
   proc chpl_serialReadWriteRectangular(f, arr, dom) {
+    chpl_serialReadWriteRectangularHelper(f, arr, dom);
+  }
+
+  proc chpl_serialReadWriteRectangularHelper(f, arr, dom) {
```

Comment:
Oh wait, I see now that you added an additional overload of the original routine to DefaultAssociative.